script/Ciel/Day 13/13_01_02_CIEL13_6_2.txt

```diff
@@ -81,37 +81,37 @@ I don't have any obligation to answer her.
 [sha:39565659b4e9397816de1ac549875873a612b821]{
 -- Page 188, Offset 35813.
 -- もう二度とあんな機会が訪れないだけだ。
-To put it simply, I'll never get that chance ever again.
+Simply put, I'll never get a chance like that again.
 }
 [sha:6db15e916350938bed4913e47e8db6ad4f28504e]{
 -- Page 189, Offset 35814.
 -- 「待って。待ってよ。待ちなさいよ。どういうコト？　情け？　情けをかけやがったの？　ピンに刺されるだけの虫のクセに、こんなに強くなったわたしを、まだ下に見るっていうの!?」
-"Wait. Hold on. Just hold on a minute. What is the meaning of this? Is this pity? Are you pitying me? Even though you're just an insect stuck on a pin, you're still looking down on me when I'm this powerful?!"
+"Wait. Hold on. Just hold on a minute. Why did you do that? Pity? Are you taking pity on me? You're just a bug that'll die when pricked with a pin, and you're looking down on me even though I'm this powerful?!"
```

Comment:
"will die when pricked with a pin" is the wrong editorialization imo. i think the original TL is more right. I think she's saying he's a pinned insect, as in he's just a little baby bitch.
![image](https://user-images.githubusercontent.com/16092757/208827562-e1021405-9dc4-467a-a365-c2b3bc8c70a9.png)